homeassistant/components/sensor/recollect_waste.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support for Recollect Waste curbside collection pickup.
+
+For more details about this platform, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.recollect_waste/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['recollect-waste==1.0.1']
+
+_LOGGER = logging.getLogger(__name__)
+ATTR_PICKUP_TYPES = 'pickup_types'
+ATTR_AREA_NAME = 'area_name'
+CONF_UPDATE_INTERVAL = 'update_interval'
+CONF_PLACE_ID = 'place_id'
+CONF_SERVICE_ID = 'service_id'
+DOMAIN = 'recollect_waste'
+ICON = 'mdi:trash-can-outline'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_PLACE_ID): cv.string,
+    vol.Required(CONF_SERVICE_ID): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(days=1)): (
```

Comment:
Why define an `update_interval` here when Home Assistant already has `scan_interval` that essentially does the same thing?

Reply:
I couldn't find much documentation on scan_interval. Many other platforms appear to set the SCAN_INTERVAL constant only. Is this sufficient?

Reply:
I realized you probably just meant use the CONF_SCAN_INTERVAL constant in HA.